app/src/main/res/values/dimens.xml

```diff
@@ -541,9 +541,9 @@
   <dimen name="promoted_story_card_layout_margin_end">8dp</dimen>
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
   <!-- The width is set to sp so that it will scale according to the device screen -->
-  <dimen name="promoted_story_card_layout_width">280sp</dimen>
+  <dimen name="promoted_story_card_layout_width">280dp</dimen>
```

Comment:
Reminder to switch this & promoted_story_card_width with their sp variants before the PR is merged.

Reply:
Changed it back to sp